From: =?utf-8?b?0L3QsNCx?= <nabijaczleweli@nabijaczleweli.xyz>
Date: Sun, 2 Jun 2024 18:04:07 +0200
Subject: Uncomment brotli simd feature (commented only to appease cargo
 install --git)

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 9239cc1..48bde5a 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -54,7 +54,7 @@ default-features = false
 
 [dependencies.brotli]
 version = "6.0"
-# features = ["simd"]  # comment this out for release
+features = ["simd"]  # comment this out for release
 
 [dependencies.iron]
 path = "vendor/iron-0.6.1"
